src/device-registry/controllers/create-device.js

```diff
@@ -326,6 +328,36 @@ const device = {
     }
   },
 
+  listAllByNearestCoordinates: async (req, res) => {
+    try {
+      const { tenant, latitude, longitude, radius } = req.query;
+
+      try {
+
+        if (!tenant || !latitude || !longitude || !radius){
```

Comment:
Could you use the AND logical operator? Assumption is that you need the use to provide all these parameters, right?

Reply:
right